csharp/extractor/Semmle.Extraction.CSharp/Entities/EventAccessor.cs

```diff
@@ -3,45 +3,46 @@
 
 namespace Semmle.Extraction.CSharp.Entities
 {
-    internal class EventAccessor : Accessor
+    internal class EventAccessor : Method
     {
-        private EventAccessor(Context cx, IMethodSymbol init)
-            : base(cx, init) { }
+        private readonly IEventSymbol @event;
+
+        private EventAccessor(Context cx, IMethodSymbol init, IEventSymbol @event)
+            : base(cx, init)
+        {
+            this.@event = @event;
+        }
 
         /// <summary>
-        /// Gets the event symbol associated with this accessor.
+        /// Gets the event symbol associated with accessor `symbol`, or `null`
+        /// if there is no associated symbol.
         /// </summary>
-        private IEventSymbol? EventSymbol => Symbol.AssociatedSymbol as IEventSymbol;
+        public static IEventSymbol? GetEventSymbol(IMethodSymbol symbol) =>
+            symbol.AssociatedSymbol as IEventSymbol;
 
         public override void Populate(TextWriter trapFile)
         {
             PopulateMethod(trapFile);
             ContainingType!.PopulateGenerics();
```

Comment:
Shouldn't we have a `PopulateModifiers(trapFile);` call here?

Reply:
Apparently not:
```
error CS1609: Modifiers cannot be placed on event accessor declarations
```